packages/backend/src/forums/forums.service.ts

```diff
@@ -27,11 +43,19 @@ export class ForumsService {
   }
 
   async getPost(id: string) {
-    const post = await this.findPost(id);
-    return post;
+    return this.findPost(id);
+  }
+
+  async getComments(id: string) {
+    const parent = await this.findPost(id);
+    const commentIds = parent.replyIds;
+    const comments = await this.ForumCommentModel.find({
+      _id: commentIds,
+    });
+    return comments as ForumComment[];
```

Comment:
Might be safer to wrap this in `DocumentType<>` in case in the future we want to do any mongoose operations on it.